devlab/provision/generate_config.sh

```diff
@@ -34,9 +34,7 @@ while read key value
 do
     value=($value)
     value=${value[1]}
-    if [[ -n ${value} ]]; then
-      sed -i "s|<${key}>|${value}|g" ${result_config}
-    fi
+    sed -i "s|<${key}>|${value}|g" ${result_config}
```

Comment:
But what is the reason to remove it? I thought the purpose of this patch was to add it instead...